Infrastructure/pom.xml

```diff
@@ -14,6 +14,22 @@
 	<version>0.0.1-SNAPSHOT</version>
 	<name>Infrastructure</name>
 
+	<properties>
+		<spring-boot.version>2.4.3</spring-boot.version>
+	</properties>
+
+	<dependencyManagement>
```

Comment:
To avoid using `spring-boot-starter-parent` in main pom.xml, we can use `spring-boot-dependencies`
See https://www.baeldung.com/spring-boot-dependency-management-custom-parent#spring-boot-without-parent-pom